android/src/main/java/com/paywallsdkreactnative/HeliumBridge.kt

```
@@ -0,0 +1,760 @@
+                // Copy file from provided URL path to helium_local
+                val sourceFile = File(java.net.URI.create(fallbackBundleUrlString))
+                if (sourceFile.exists()) {
+                    sourceFile.copyTo(destinationFile, overwrite = true)
+                    fallbackBundleName = "helium-fallback.json"
+                }
+            } else if (fallbackBundleString != null) {
+                // Write fallback bundle string to file
+                destinationFile.writeText(fallbackBundleString)
+                fallbackBundleName = "helium-fallback.json"
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)